blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""This module contains models for the Blog application."""
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Model representing a blog post."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish post by setting publish datetime."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return post title as string representation."""
```

Comment:
No, it returns a string representation of the post. The fact that it uses the title for this, it an implementation detail and not something to put in a docstring title.

Reply:
Edited.

Reply:
Marking as resolved.